### PR TITLE
Adding "align-items" (need testing)

### PR DESCRIPTION
**Pull request by XxFlameCatxX**
Depiste this update it need testing, i have no idea how responsive is the joystream, so it's possibly the responsivity. But i will need it being tested with different resolutions. I can't test it wihout having a bigger screen. SO you guys could test it for me, i don't know if it will work

Problem: https://github.com/Joystream/pioneer/issues/3382

### CODE REVIEW

**Review by oleksanderkorn — CHANGES_REQUESTED** [review]

**Comment by oleksanderkorn** [comment]
@XxFlameCatxX it looks like the `top` posititon of `AccountLocksWrapper` can be adjusted to add more space between icons wrapper and account balance, this works for me, wdyt?
```
export const AccountLocksWrapper = styled(ColumnGapBlock)`
  position: absolute;
  top: 20px;
  align-items: center;
`
```

image

File: .gitignore [hunk 1]
```diff
@@ -6,4 +6,4 @@ build-storybook
 *.log
 .DS_Store
 packages/ui/.env
-packages/ui/src/services/i18n/utils/csv
+packages/ui/src/services/i18n/utils/csv
```
Comment (oleksanderkorn) [inline .gitignore:9]:
```suggestion
packages/ui/src/services/i18n/utils/csv

```

**Review by oleksanderkorn — CHANGES_REQUESTED** [review]
add new line

**Review by oleksanderkorn — APPROVED** [review]

**Comment by traumschule** [comment]
Failing check in [DeleteThreadModal](https://github.com/Joystream/pioneer/pull/3803/checks#step:7:55) seems unrelated.

File: packages/ui/src/accounts/components/AccountItem/components/styles.ts [hunk 2]
```diff
@@ -72,6 +72,7 @@ export const LocksButtons = styled.div`
   align-self: end;
   display: flex;
   column-gap: 10px;
+  align-items: center;
```
Comment (oleksanderkorn) [inline packages/ui/src/accounts/components/AccountItem/components/styles.ts:75]: I thought i wrote this before but could not find my comment:
This  line can be removed, has no impact at all.

**Review by oleksanderkorn — CHANGES_REQUESTED** [review]

File: packages/ui/src/accounts/components/AccountItem/components/styles.ts (hunk 2 above)
Comment (traumschule) [reply packages/ui/src/accounts/components/AccountItem/components/styles.ts:75]: Had the same feeling. Likely because of commit overlap with #3828.

**Comment by XxFlameCatxX** [comment]
closed

**Pull request closed without merging**